rts/lra/test/README.adoc

```diff
@@ -1,57 +1,63 @@
+and from the client(s).
+All tests are executed with help of https://arquillian.org[Arquillian] platform. In some tests, Arquillian is coupled
+with maven plugin to start the runtime and deploy lra-coordinator.
+
 The `lra/test` module contains the following submodules
 
-* link:./arquillian-extension/[`arquillian-extension`] contains set of
-  Arquillian extension classes which makes possible to execute the test modules
-  on supported runtimes
+* link:./arquillian-extension/[`arquillian-extension`] contains Arquillian extension classes
+  which make possible to execute the test modules on supported runtimes
 * link:./basic[`basic`] contains integration tests which are written for testing
   the Narayana features of the
   https://github.com/eclipse/microprofile-lra[LRA specification]
-* link:./tck[`tck`] is a module which is responsible for execution of
+* link:./tck[`tck`] is a module which is responsible for the execution of
   https://github.com/eclipse/microprofile-lra/tree/master/tck[LRA TCK tests]
   with Narayana supported runtimes
-
-All tests are executed with help of https://arquillian.org[Arquillian] platform.
-Arquillian is responsible to launch the runtime and makes possible to execute
-the tests.
+* link:./crash[`crash`] hosts tests that simulate a crash of either lra-coordinator or
+  lra-participant or both. So far, only Wildfly is considered in this module as Quarkus
+  does not have an Arquillian adapter to run unmanaged tests
 
 == How to run integration tests
 
-The itegration tests which should be run on the application runtime
-are configured to be started with https://maven.apache.org/surefire/maven-failsafe-plugin/[Maven Failsafe Plugin].
+Integration tests are tested with
+https://maven.apache.org/surefire/maven-failsafe-plugin/[Maven Failsafe Plugin], using the target 'verify'.
 
-To run the tests the target `verify` has to be used.
-Single test with `-Dit.test=...`.
-Debug the Maven test execution with `-Dmaven.failsafe.debug`.
+Single test can be executed with `-Dit.test=...`
+To use Failsafe in debug mode, use `-Dmaven.failsafe.debug`
 
-The Failsafe Plugin link:../pom.xml#114[is configured] to run tests with names
+The Failsafe Plugin link:../pom.xml[is configured] to run tests with names
 `*IT.java`, `*Tests.java` and `*TestCase.java`.
 
 === Application runtimes
 
 The module is configured to run two runtimes - https://www.wildfly.org[WildFly]
-and link:https://quarkus.io[Quarkus]/link:https://thorntail.io[Thorntail].
+and link:https://quarkus.io[Quarkus].
 
-The **WildFly** runtime deploys the coordinator and the application with the client
-library at the same application server. The coordinator is deployed as
-a separated `war` archive, and the same the applications which packs together
-the LRA client library.
+When the **WildFly** runtime is used (through the 'arq' profile), two instances of Wildfly are initiated:
```

Comment:
So there is a profile called `arq` that runs the integration tests against WildFly?
And there is another one called `arq.thorntail` which runs against what?
And what does ARQ_THORN do?
And do we use ARQ_PROF=arqIPv6 in the LRA subsystem at all?

Reply:
`arq.thorntail` could also be renamed to `arq.quarkus`.
I do not think we use ARQ_PROF=arqIPv6 ATM. Should we open a new JIRA about this?

Reply:
The question is not only for my benefit, it is for the benefit of any community member that wants to run our test suite.

Reply:
I will rename `arq.thorntail` to `arq.quarkus`. What about the new JIRA for ARQ_PROF=arqIFv6? I think it would be good to have a discussion about this in a separate JIRA/PR, WDYT @mmusgrov?

Reply:
Yes, what I remember we are not using the ipv6 profile for LRA testing. To me the ipv6 question would be good to be opened as a new issue.

Reply:
+1 Thanks Ondra

Reply:
It's very complicated but fair enough, it's okay to resolve this conversation now.